org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/DeviceResponse.java

```diff
@@ -156,21 +160,21 @@ private static void processScannables(DeviceRequest request, IScannableDeviceSer
 	private static void merge(DeviceInformation<?> info, IScannable<?> device) throws Exception {
 		info.setLevel(device.getLevel());
 		info.setUnit(device.getUnit());
-        info.setUpper(device.getMaximum());	
+        info.setUpper(device.getMaximum());
         info.setLower(device.getMinimum());
         info.setPermittedValues(device.getPermittedValues());
         info.setActivated(device.isActivated());
         info.setMonitorRole(device.getMonitorRole());
  	}
 
 	private static void processRunnables(DeviceRequest request, IRunnableDeviceService dservice) throws Exception {

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 32 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)